src/vs/workbench/contrib/chat/browser/chatQuotasService.ts

```diff
@@ -264,10 +264,10 @@ export class ChatQuotasStatusBarEntry extends Disposable implements IWorkbenchCo
 
 export function quotaToButtonMessage({ chatQuotaExceeded, completionsQuotaExceeded }: { chatQuotaExceeded: boolean; completionsQuotaExceeded: boolean }): string {
 	if (chatQuotaExceeded && !completionsQuotaExceeded) {
-		return localize('chatQuotaExceededButton', "You've reached your monthly chat messages limit, click for details");
+		return localize('chatQuotaExceededButton', "Monthly chat messages limit reached, click for details");
```

Comment:
I think the grammatically correct version should be:

`Monthly chat message limit reached. Click for details.`

Reply:
Yes sir

Reply:
Sounds good, thanks!